controller/handler_edge_ctrl/extend_enrollment_verify.go

```diff
@@ -82,7 +84,25 @@ func (h *extendEnrollmentVerifyHandler) HandleReceive(msg *channel.Message, ch c
 				return
 			}
 
-			if *edgeRouter.UnverifiedCertPem != verifyMsg.ClientCertPem {
+			submittedCerts := nfpem.PemStringToCertificates(verifyMsg.ClientCertPem)
+
+			if len(submittedCerts) == 0 {
+				h.respond(&edge_ctrl_pb.Error{
+					Code:    "INVALID_CLIENT_PEM",
+					Message: "request did not contain a parsable PEM certificate",
+				}, msg, ch)
```

Comment:
should these return like hte others below do? (line 110, 120)

Reply:
Yes they should.